TSIS2.WebResources/Webresources/js/QuestionnaireVersion.js

```diff
@@ -4,10 +4,11 @@ var ROM;
     var QuestionnaireVersion;
     (function (QuestionnaireVersion) {
         function onLoad(eContext) {
+            var _a;
             setNotificationMessage(eContext);
             // Get formContext
             var Form = eContext.getFormContext();
-            var surveyDefinition = Form.getAttribute("ts_questionnairedefinition").getValue();
+            var surveyDefinition = (_a = Form.getAttribute("ts_questionnairedefinition")) === null || _a === void 0 ? void 0 : _a.getValue();
```

Comment:
Some funky intellisense fix here?

Reply:
Oh that would be from trying [the ? operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html) to check for undefined in the typescript, but it doesn't seem necessary. [Removed it](https://github.com/tc-ca/ROM.TSIS2.Webresources/pull/197/commits/f37a2e1b485616bd5c8e57107de3054fb5c11b36)

Reply:
Makes sense, thanks!